examples/workflows/compare-release/README.md

```
@@ -0,0 +1,64 @@
+CLI). See that plan's "Known gaps" table for the rest of this curated set,
+not yet built.
+
+## The project
+
+A tiny shared library, `mathutils`, ships two releases:
+
+| | v1 | v2 |
+|---|---|---|
+| `mathutils.h` | `add`, `subtract`, `multiply` | `add`, `multiply` — **`subtract` removed** |
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)